Dashboard/app/js/lib/views/views.js

```diff
@@ -555,11 +555,10 @@ FLOW.DateField2 = Ember.TextField.extend({
     this._super();
 
     this.$().datepicker({
-      dateFormat: 'yy/mm/dd',
+      dateFormat: 'yy-mm-dd',
```

Comment:
For some reason the date picked popup doesn't work under the inspect data tab when editing date responses